website/docs/docs/build/sl-getting-started.md

```diff
@@ -35,7 +38,7 @@ To experience the power of a universal [dbt Semantic Layer](/docs/use-dbt-semant
 -  Create a successful run in the environment where you configure the Semantic Layer. 
    - **Note:** Semantic Layer currently supports the Deployment environment for querying. (_development querying experience coming soon_)
 - Set up the [Semantic Layer API](/docs/dbt-cloud-apis/sl-api-overview) in the integrated tool to import metric definitions. 
-  - **Note:** dbt Core or Developer accounts can only query data manually using the [MetricFlow CLI](/docs/build/metricflow-cli) and SQL. To dynamically query metrics using external tools, you must have a dbt Cloud [Team or Enterprise](https://www.getdbt.com/pricing/) account with access to the Semantic Layer API.<br />
+  - **Note:** To access the API and query metrics in downstream tools, you must have a dbt Cloud [Team or Enterprise](https://www.getdbt.com/pricing/) account. dbt Core or Developer accounts can define metrics using the [MetricFlow CLI](/docs/build/metricflow-cli).<br />
```

Comment:
The MetricFlow CLI installation instructions state: You can install MetricFlow using PyPI as an extension of your `dbt adapter in the CLI`." Will that have to do additional installations outside of just MetricFlow to get this working with a dbt Cloud Developer account? If so, we may need to add that to the CLI page as I don't see any mentions of it working with a cloud account.

Reply:
the cli doesn't work in cloud so we don't need to add the info. but maybe i can add 'ide' to clarify.